### ISSUE TITLE
Issue when starting the docker

### ISSUE THREAD

**Issue by ahiguero** [jacobalberty/unifi-docker#26]
After shutting down and restarting the docker this is what i get

Starting unifi controller service.                                                                                               
WARN: unifi service process ended without being singaled? Check for errors in /var/log/unifi.

**Comment by jacobalberty** [comment]
Whats in the logfile? You can retrieve all 4 with the following commands
```

docker container cp <container>:/var/log/unifi/unifi.err.log ./
docker container cp <container>:/var/log/unifi/unifi.out.log ./
docker container cp <container>:/var/log/unifi/server.log ./
docker container cp <container>:/var/log/unifi/mongod.log ./

```

**Comment by wutiarn** [comment]
I think this happens after killing container (docker kill) and restarting it again. Personally I solved this issue by cleaning up content of /var/run/unifi (inside of container) every time before controller starts.

See https://gist.github.com/wutiarn/c033d0c494ce5d42620efb80bae5c92b

I know, that's pretty straightforward, but it works perfectly.

**Comment by jacobalberty** [comment]
@ahiguero , any chance you could try clearing the contents of /var/run/unifi  http://stackoverflow.com/questions/32750748/how-to-edit-files-in-stopped-not-starting-docker-container has a decent description of how to accomplish this I prefer the second method but do whichever you are more comfortable with. It shouldn't be too difficult for me to integrate @wutiarn's fix into the main image.

**Comment by wutiarn** [comment]
Clearing /var/run/unifi once would help only until next "hard" container restart. More convenient way to hotfix it right now (without modifying image) for @jacobalberty is to recreate container. If, of course, /var/lib/unifi is safe (binded to host path or persistent volume).

**Comment by jacobalberty** [comment]
@wutiarn  yeah I can simply add the clean /var/run/unifi to unifi.sh, I don't see any downside to doing that I just want to confirm its the same thing @ahiguero  is running into.

**Comment by jacobalberty** [comment]
@wutiarn  try the jacobalberty/unifi:wutiarn branch, it should do everything your fix does, im a bit curious which exact file it is though, theres some stuff relating to the updates that might not need to be removed, im guessing just unifi.pid needs to be removed

**Comment by wutiarn** [comment]
> im guessing just unifi.pid needs to be removed

Yes, you're absolutely right. I've managed to reproduce this issue with the original image (unifi5 branch) and manual removal of unifi.pid helped.

**Comment by jacobalberty** [comment]
This should be fixed in master now, I retagged 5.4.11 to include it as well so the following tags will be fixed `unifi5` `latest` `5.4.11`
